src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: false,
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+});
+
+// console.log(categoriesSlice);
```

Comment:
- [ ] Kindly refactor your categories reducer so that it returns 'Under construction'.

- [ ] Kindly remove commented code on line 13 also.